server/application/application.go

```diff
@@ -1013,7 +1072,7 @@ func (s *Server) getCachedAppState(ctx context.Context, a *appv1.Application, ge
 	return err
 }
 
-func (s *Server) GetAppResources(ctx context.Context, a *appv1.Application) (*appv1.ApplicationTree, error) {
+func (s *Server) getAppResources(ctx context.Context, a *appv1.Application) (*appv1.ApplicationTree, error) {
```

Comment:
What is the reason to make this a private method? It seems an useful method to have available.

Reply:
Each time I see it, I'm confused why there's no RBAC check. It's because this function isn't used externally. Making it private just makes it a bit clearer why there isn't an RBAC check.